src/app/view/modules/framework/framework-for-listings/framework-for-listings.component.ts

```diff
@@ -103,9 +129,9 @@ export class FrameworkForListingsComponent implements OnInit, OnChanges {
 
     ngOnInit() {
         // bad hack to get the project admin information
-        this.loggedInAdmin = (sessionStorage.getItem('admin') !== null);
-        // end of bad hack. TODO: we have to find a better solution
-
+        if (localStorage.getItem('currentUser') !== null) {
```

Comment:
Are we using localStorage or sessionStorage?

Reply:
We're using sessionStorage for temporary information e.g. in project admin we store current project, current project members etc.
The localStorage is used to store the information about the logged in user called currentUser.